packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -222,6 +222,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{
           softMin: customConfig.axisSoftMin,
           softMax: customConfig.axisSoftMax,
           centeredZero: customConfig.axisCenteredZero,
+          range: customConfig.stacking?.mode === StackingMode.Percent ? [0, 1] : undefined,
```

Comment:
this will cause us to ignore any soft or hard min/max y settings when `StackingMode.Percent`, but that's likely fine.